### ISSUE TITLE
ValueError: bad input shape () in basic_usage.ipynb example

### ISSUE THREAD

**Issue by PeerHerholz** [mvdoc/meegnobis#6]
Hi @mvdoc,

I was just trying the [example notebook](https://github.com/mvdoc/meegnobis/blob/master/examples/basic_usage.ipynb) and run into an error in this line:
`rdm, target_pairs = compute_temporal_rdm(epoch, targets, cv=cv, metric='euclidean', cv_normalize_noise='epoch', mean_groups=True)`.

The error is `ValueError: bad input shape ()` and refers to `sklearn` [here](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/utils/validation.py#L781) and [here](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/preprocessing/_label.py#L289), being utilized in `meegnobis` [here](https://github.com/mvdoc/meegnobis/blob/master/meegnobis/rsa.py#L364).

Do you perhaps have any pointers on how to tackle this?

Best, Peer

**Comment by mvdoc** [comment]
Hi @PeerHerholz, I think sklearn added extra checks on LabelEncoder and didn't allow passing an int anymore. Indeed some tests were failing. I pushed changes and commit 1952689e53b9b4460fd54fb762b1a980e401fc4d should fix the problem. Let me know if you still get the same issue.

Also note that I abandoned this project and for lack of bandwidth (and change in research interests) I'm not actively developing it.
